setup/gcp/installer/util/constants.py

```diff
@@ -35,6 +35,19 @@ class FirewallRuleDirection(Enum):
 DEFAULT_BUCKET_FMT_V1 = 'gs://{}-data-{}'
 DEFAULT_BUCKET_FMT_V2 = 'gs://{}-{}-data-{}'
 
+FORSETI_V1_RULE_FILES = [
```

Comment:
Is it possible a user could have iam_rules.yaml and iam_rules2.yaml? Is it better to match for *.yaml?